system-set-up.sh

```diff
@@ -11,4 +11,18 @@ echo "Running system setup ruby script"
 ruby system-set-up.rb
 echo "system setup ruby script finished"
 
-echo "Done!"
+# SDK Man stuff
+curl -s https://get.sdkman.io | bash
+source "/Users/$(whoami)/.sdkman/bin/sdkman-init.sh"
```

Comment:
Maybe it's easy to use the home character next time. `~/.sdkman/bin/sdkman-init.sh`